docs/architecture/policy_credentialtype_scope.md

```diff
@@ -0,0 +1,153 @@
+  type discriminates which framework agreement use case the credential attests to. It is **not** a separate
+  VerifiableCredential type! For
+  example: [PCF Use case credential](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/3.%20Verifiable%20Credentials/CX-Credentials/Standardized%20CX-Credential.md#pcf-use-case-credential).
+  Specifically, `subject.type = PcfCredential`. Also referred to as: use case
+
+- Contract version: refers to the `credentialSubject.contractVersion` property and is expected to be
+  a [SemVer string](https://semver.org/). The `contractVersion` discriminates which framework agreement use case version
+  the credential attests to. Also referred to as a `framework agreement version`.
+
+- Credential version: a credential version does not exist as a first-class concept.
+
+## Referencing credentials in policies
+
+Please check the
+syntax [here](https://github.com/eclipse-tractusx/ssi-docu/blob/main/docs/architecture/cx-3-2/edc/policy.definitions.md)
+
+## Mapping Framework credentials
+
+Automatic mapping policy expression to credential type is possible assuming all framework credentials follow the same
+schema. If all Framework credentials conform to the same schema, the evaluation of the policy (= the evaluation
+function) can be generic.
+
+In an actual example a policy constraint, requiring the "pcf" credential in version 0.4.2, would look like this:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.pcf",
+    "operator": "eq",
+    "rightOperand": "active:0.4.2"
+  }
+}
+```
+
+If a specific contract version is referenced in the policy constraint, like in the example above, it follows that the
+policy must be **updated**, if version 43 of the `pcf` credential is released, and the policy should reference it.
+
+If no contract version is specified in the policy constraint, then the policy **need not be updated**.
+
+### Correlation with the credential type
+
+A use case type can be constructed easily by simple string manipulation:
+
+- remove `FrameworkAgreement.`
+- capitalize the remainder (i.e. `pcf --> Pcf`)
+- append the `Credential` literal
+
+which would result in the term `PcfCredential`. In addition, parsing the `rightOperand` would yield the (optional)
+version identifier. The version identifier is processed separately, e.g. in a database query.
+
+Formally, the use case type can be constructed using the BNF below.
+
+### Converting to a scope string
+
+The conversion from use case type to scope string and vice versa can be performed without any static mapping, but by
+relatively simple string manipulation:
+
+- start with the (dataspace-specific) scope prefix (`org.eclipse.tractusx.vc.type`)
+- append a `:`
+- append the use case type
+- [optional]: append a `_` and the version identifier (e.g. `0.4.2`)
+- append a `:`
+- append the operator (`read`, `write`, `*`)
+
```

Comment:
I know this is nothing new - but need some context explanation to be able to fully evaluate this. 
'read' stands for "access policy" I assume? If not, what what does 'read' actually stands for?

Reply:
no. `read` means "give me credential" whereas `write` means "here you have credential". The earlier is relevant for presentation flow (the verifier wants to have the holder's VC), whereas the latter is relevant for the issuance flow (issuer writes VC into holder's CredentialService).
The scope syntax is the same though.